packages/nimble-components/src/anchor-button/tests/anchor-button-matrix.stories.ts

```diff
@@ -41,17 +41,17 @@ const partVisibilityStates = [
     [true, true, true],
     [false, true, true]
 ] as const;
-type PartVisibilityState = typeof partVisibilityStates[number];
+type PartVisibilityState = (typeof partVisibilityStates)[number];
```

Comment:
This is a style change enforced by prettier https://github.com/prettier/prettier/issues/14162#issuecomment-1380095817